config.js

```diff
@@ -1,3 +1,5 @@
+require('dotenv').config({path: './.env'});
```

Comment:
@Garethrehal @mikemonteith make sure to change the path to .env_staging here to get your staging setup updating